cps/helper.py

```diff
@@ -239,7 +245,10 @@ def get_valid_filename(value, replace_whitespace=True):
         value=value.replace(u'ß',u'ss')
         value = unicodedata.normalize('NFKD', value)
         re_slugify = re.compile('[\W\s-]', re.UNICODE)
-        value = unicode(re_slugify.sub('', value).strip())
+        try:
+            value = str(re_slugify.sub('', value).strip())
+        except: #will exception on Python2.7
```

Comment:
To catch any exception use the ```except Exception```, instead of ```except BaseException``` or just ```except```.
Indicate the most specific exception type in ```except``` block.
For example:
```python
# Bad
try:
    mapping[key]
except Exception:
    ...

# Better
try:
    mapping[key]
except KeyError:
    ...
```